setup.py

```diff
@@ -3,18 +3,28 @@
 #===================================================================================================
 # Imports
 #===================================================================================================
-from distutils.core import setup
+from setuptools import setup
 from vmpooler_client.version import version
 
+classifiers = [
```

Comment:
Being picky here:

I think that the classifiers should be under a comment header "Globals". The ```classifiers``` variable name should be all caps to indicate that it is a global.